### PR TITLE
feat: Add data because of dir change

### PR DESCRIPTION
**Pull request by jeffshuai**
Add data because of dir change

Log: Add data because of dir change

### CODE REVIEW

**Review by Groveer — COMMENTED** [review]
+1

**Review by toberyan — APPROVED** [review]

File: tests/CMakeLists.txt [hunk 1]
```diff
@@ -19,6 +19,12 @@ add_definitions(-DPREFIX="${CMAKE_PREFIX_PATH}")
 add_definitions(-DOS_VERSION_TEST_FILE="/tmp/etc/os-version")
 find_package(Qt5 REQUIRED COMPONENTS Core)
 find_package(Qt5 REQUIRED COMPONENTS DBus)
+find_package(PkgConfig REQUIRED)
+pkg_search_module(LIB_NL3 REQUIRED libnl-3.0)
+pkg_search_module(LIB_NL3_ROUTE REQUIRED libnl-route-3.0)
+
+include_directories(${LIB_NL3_INCLUDE_DIRS})
```
Comment (asterwyx) [inline tests/CMakeLists.txt:26]: 推荐使用PkgConfig的IMPORTED_TARGET方式,直接使用target_link_libraries,避免使用include_directories

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -27,6 +34,8 @@ target_link_libraries(
     ${LIB_NAME} PUBLIC
     Qt5::Core
     Qt5::DBus
+    ${LIB_NL3_LIBRARIES}
+    ${LIB_NL3_ROUTE_LIBRARIES}
```
Comment (asterwyx) [inline src/CMakeLists.txt:38]: 同tests

File: src/CMakeLists.txt [hunk 3]
```diff
@@ -14,9 +14,16 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
...
+find_package(PkgConfig REQUIRED)
 find_package(Qt5 REQUIRED COMPONENTS Core)
 find_package(Qt5 REQUIRED COMPONENTS DBus)
 
+
+pkg_search_module(LIB_NL3 REQUIRED libnl-3.0)
+pkg_search_module(LIB_NL3_ROUTE REQUIRED libnl-route-3.0)
+
+include_directories(${LIB_NL3_INCLUDE_DIRS})
+include_directories(${LIB_NL3_ROUTE_INCLUDE_DIRS})
```
Comment (asterwyx) [inline src/CMakeLists.txt:26]: 同tests

File: src/CMakeLists.txt [hunk 4]
```diff
@@ -14,9 +14,16 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 add_definitions(-DLIBDTKDEVICE_LIBRARY)
 
 # find_package
+find_package(PkgConfig REQUIRED)
 find_package(Qt5 REQUIRED COMPONENTS Core)
 find_package(Qt5 REQUIRED COMPONENTS DBus)
 
+
+pkg_search_module(LIB_NL3 REQUIRED libnl-3.0)
+pkg_search_module(LIB_NL3_ROUTE REQUIRED libnl-route-3.0)
```
Comment (asterwyx) [inline src/CMakeLists.txt:23]: 同tests

**Review by asterwyx — CHANGES_REQUESTED** [review]

**Pull request merged**